manifests/cluster-install/workflow-controller-rbac/workflow-controller-role.yaml

```diff
@@ -17,4 +17,5 @@ rules:
       - secrets
     verbs:
       - get
+      - watch
```

Comment:
This introduces a new requirement. @jessesuen Would you be okay with this addition?

Reply:
This would be equivalent to adding `list`, which is expanding the scope such that the controller see all secrets in the namespace. (Actually, I'm now wondering why `list` wasn't added).

Is this Role used only for the controller's namespace (e.g. `argo`)?